tornado/test/simple_httpclient_test.py

```diff
@@ -322,12 +330,11 @@ def test_options_request(self):
     def test_no_content(self):
         response = self.fetch("/no_content")
         self.assertEqual(response.code, 204)
-        # 204 status doesn't need a content-length, but tornado will
-        # add a zero content-length anyway.
+        # 204 status shouldn't have a content-length
         #
         # A test without a content-length header is included below
         # in HTTP204NoContentTestCase.
```

Comment:
Yeah, I was a little unsure about how best to organize it. Wasn't this test already covering that by doing its followup request for `"/no_content?error=1"`? Or do you mean to move that part of the test into the other test case (and rename I suppose)?